hail/python/hail/experimental/datasets.py

```diff
@@ -2,56 +2,63 @@
 import json
 import hail as hl
 
-
-def load_dataset(dataset_name,
-                 reference_genome,
-                 config_file='gs://hail-datasets/hail_datasets.config.json'):
-    """Load a Hail-formatted genetic dataset.
+def load_dataset(name,
+                 version,
+                 reference_genome):
+    """Load a genetic dataset from Hail's repository.
 
     Example
     -------
 
-    >>> # Load 1000 Genomes chromosome X MatrixTable with GRCh38 coordinates
-    >>> mt_1kg = hl.experimental.load_dataset('1000_genomes_phase3_chrX',   # doctest: +SKIP
+    >>> # Load 1000 Genomes MatrixTable with GRCh38 coordinates
+    >>> mt_1kg = hl.experimental.load_dataset(name='1000_genomes',   # doctest: +SKIP
+    ...                                       version='phase3',
     ...                                       reference_genome='GRCh38')
 
     Parameters
     ----------
-    dataset_name : :obj:`str`
+    name : :obj:`str`
         Name of the dataset to load.
+    version : :obj:`str`
+        Version of the named dataset to load
+        (see available versions in documentation).
     reference_genome : `GRCh37` or `GRCh38`
         Reference genome build.
-    config_file : :obj:`str`, optional
-        Path of the datasets configuration file.
-        Leave as default if running on Google Cloud Platform.
 
     Returns
     -------
     :class:`.Table` or :class:`.MatrixTable`"""
 
-    with hl.hadoop_open(config_file, 'r') as f:
-        config = json.load(f)
+    with hl.hadoop_open('gs://hail-datasets/datasets.json', 'r') as f:
+        datasets = json.load(f)
 
-    builds = [{'path': x['path'], 'reference_genome': x['reference_genome']}
-              for x in config if x['name'] == dataset_name]
+    names = set([dataset['name'] for dataset in datasets])
+    if name not in names:
+        raise ValueError('"{}" is not a dataset available in the repository.'.format(name))
```

Comment:
calling `repr(name)` means you don't need to explicitly include quotes, and also means this is resilient to the user passing in something with newlines/tabs etc